cmd/argocd/commands/admin/admin.go

```diff
@@ -161,22 +169,12 @@ func specsEqual(left, right unstructured.Unstructured) bool {
 	return false
 }
 
-type argocdAdditonalNamespaces struct {
-	applicationNamespaces    []string
-	applicationsetNamespaces []string
-}
-
-const (
-	applicationsetNamespacesCmdParamsKey = "applicationsetcontroller.namespaces"
-	applicationNamespacesCmdParamsKey    = "application.namespaces"
-)
-
 // Get additional namespaces from argocd-cmd-params
-func getAdditionalNamespaces(ctx context.Context, argocdClientsets *argoCDClientsets) *argocdAdditonalNamespaces {
+func getAdditionalNamespaces(ctx context.Context, configMapsClient dynamic.ResourceInterface) *argocdAdditonalNamespaces {
```

Comment:
why you don't use `argocdClientsets`

Reply:
because only the configMap client is required.